app/src/main/java/org/thoughtcrime/securesms/onboarding/landing/Landing.kt

```diff
@@ -129,10 +131,21 @@ internal fun LandingScreen(
                     MESSAGES.take(count),
                     key = { it.stringId }
                 ) { item ->
-                    AnimateMessageText(
-                        stringResource(item.stringId),
-                        item.isOutgoing
-                    )
+                    // Perform string substitution in the bubbles that require it
```

Comment:
There is no reason to have the composable defined twice in the if. We only want that one composable but it is its `text` property that needs to changed based on some logic.
So `AnimateMessageText` can be defined once and passed the appropriate content

Reply:
Done